cmd/api/v1/volume.go

```diff
@@ -248,3 +269,76 @@ func connectedHosts(vol *apiv1.Volume) []string {
 
 	return nodes
 }
+
+func (v *volume) updateFromCLI(args []string) (*apiv1.VolumeServiceUpdateRequest, error) {
```

Comment:
As there is some complexity here, I think it would be great to have a test for the volume commands. If you need help with it, I can also write it.

Reply:
Tests done :heavy_check_mark: